runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -115,11 +115,22 @@ default String getHiddenItems()
 		return "";
 	}
 
+	@ConfigItem(
+			keyName = "defaultColor",
+			name = "Default items color",
+			description = "Configures the color for default, non-highlighted items",
+			position = 8
```

Comment:
use only 1 tab to indent, not 2

Reply:
oops, I'm not sure how I didn't see that